components/forms/_checkboxes/checkbox.scss

```diff
@@ -106,3 +106,18 @@
     top: 6px;
   }
 }
+.msds-checkbox.msds-checkbox-inline {
```

Comment:
If we follow BEM with my last comment regarding the modifier, I would move this code to line 107 and use SCSS to structure the code. 
I would set it up more or less as follows:
.msds-checkbox{
 ...
&--inline{
 .msds-checkbox__label {
    display: inline;
    margin-block-start: 0;
    margin-block-end: 0;
    vertical-align: top;
    &:before {
      vertical-align: bottom;
      margin-right: 6px;
    }
    p {
      display: inline;
    }
  }
}
}